lib/liquider/ast.rb

```diff
@@ -67,5 +77,9 @@ def visit(visitor)
   NullNode = Node.new_type(:null) do
     def visit(visitor)
     end
+
+    def op?
```

Comment:
This is only so we can avoid empty `else` after each `if` that don't have their own `else`